Lib/glyphsLib/builder/builders.py

```diff
@@ -387,8 +387,14 @@ def _valid_designspace(self, designspace):
         """Make sure that the user-provided designspace has loaded fonts and
         that names are the same as those from the UFOs.
         """
-        # TODO: (jany) really make a copy to avoid modifying the original object
-        copy = designspace
+        import copy
+
+        copy = copy.deepcopy(designspace)
```

Comment:
if the sources have a 'font' attached with a defcon.Font, this deepcopy will probably fail
https://github.com/typesupply/defcon/issues/210

Reply:
also not a pretty choice for a local variable name that masks the just imported `copy` module